frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigCard.js

```diff
@@ -35,7 +33,7 @@ export default function ConfigCard(card) {
           Alert Configuration for "{ card.data }"
         </Typography>
         <Typography>
-          When sent an alert regarding "{ card.data }", you will also be sent data for "{ card.rData }".
+          When sent an alert regarding "{ card.data }", you will also be sent data for "{ card.relatedData }".
```

Comment:
can you confirm the display is still correct? I think before there might've been a bug wrt the assumptions of `rData` vs `relatedData` so just want to double check

Reply:
yes, now it's correct - this is a bug fix because `rData` is no longer what was used to name the related data, it's actually `relatedData`.